web/tests/lib/index.js

```diff
@@ -38,6 +39,9 @@ require("@babel/register")({
     root: path.resolve(__dirname, "../.."),
 });
 
+// Ensure that startTransaction and friends are available at runtime
+Sentry.addTracingExtensions();
```

Comment:
Can you check if this materially impacts test suite runtime (probably easiest verified running them without parallelism enabled)? I would hope not, but since I don't know what these extensions are, I figure it's worth checking.

Reply:
If there's any change, it's very much in the noise:
```
$ hyperfine -L commit 202cae7a,853e41db --setup 'git checkout {commit}' -n '{commit}' './tools/test-js-with-node --parallel=1'
Benchmark 1: 202cae7a (with addTracingExtentions)
  Time (mean ± σ):     13.700 s ±  0.365 s    [User: 15.874 s, System: 0.848 s]
  Range (min … max):   13.257 s … 14.348 s    10 runs

Benchmark 2: 853e41db (before addTracingExtensions)
  Time (mean ± σ):     13.920 s ±  0.412 s    [User: 16.243 s, System: 0.806 s]
  Range (min … max):   13.316 s … 14.559 s    10 runs

Summary
  '202cae7a (with addTracingExtensions)' ran
    1.02 ± 0.04 times faster than '853e41db (before addTracingExtensions)'
```

Reply:
Yeah that seems fine.